README.md

```diff
@@ -268,6 +268,7 @@ For each connections defined in such a manner DANM will provision exactly one in
 In addition to simply invoking other CNI libraries to set-up network connections, Pod's can even influence the way their interfaces are created to a certain extent.
 For example Pods can ask DANM to provision L3 IP addresses to their IPVLAN or SRI-OV interfaces dnyamically, statically, or not at all!
 Or, creation of policy-based L3 IP routes into their network namespace is also a supported by the solution.
+Note: if the Pod annotation is empty (no danmnet connection is defined), DANM will fall back to use a default network definition, if created. The danmnet object in the target K8S namespace must use the name: "default". It can have any network type (either delegated or ipvlan). Naturally, user in this case cannot specify any further property for the Pod (i.e. static IP address). This way, the user is able to use unmodified manifest files (i.e. community maintained Helm charts or Pods created by K8S operators).
```

Comment:
could we move this to one paragraph higher, i.e. starting from line 267?

It would make sense to talk about the fallback option right after we detailed how can a Pod define network connections

Reply:
Done.